### PR TITLE
chore: minor cleanup

### PR DESCRIPTION
**Pull request by descorp**
### Changes

* remove public helper for UIViewController
* make `preferredBarTintColor` optional and default value to **nil**
* add 1sec delay for UIKit Demo Alert presentation

### CODE REVIEW

File: Adyen.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -3785,7 +3785,6 @@
 				F924C079246C38E0006DDAB8 /* SecuredViewController.swift in Sources */,
 				E7D5311A2446F525000046B4 /* FormSeparatorItem.swift in Sources */,
 				E9E3DAFD2225485400697074 /* URLSessionHelpers.swift in Sources */,
-				F926D53523F68FD300D058D3 /* QiwiWalletDetails.swift in Sources */,
```
Comment (descorp) [inline Adyen.xcodeproj/project.pbxproj:3788]: This is some weird artefact..

**Review by mohammedDehairy — DISMISSED** [review]

File: Adyen/Helpers/UIViewControllerHelpers.swift [hunk 2]
```diff
@@ -39,12 +24,13 @@ public extension AdyenScope where Base: UIViewController {
 
     /// :nodoc:
     func present(_ viewController: UIViewController, animated: Bool, completion: (() -> Void)? = nil) {
+        let topPresenter = self.topPresenter
         if topPresenter.isBeingPresented || topPresenter.isBeingDismissed {
-            DispatchQueue.main.asyncAfter(deadline: .now() + .milliseconds(10)) {
-                self.present(viewController, animated: animated, completion: completion)
+            DispatchQueue.main.asyncAfter(deadline: .now() + 1) {
```
Comment (descorp) [inline Adyen/Helpers/UIViewControllerHelpers.swift:29]: 10 milisec could be not enough. This is used for our Demo only, so should be fine

File: Adyen/Helpers/UIViewControllerHelpers.swift [hunk 3]
```diff
@@ -24,12 +24,13 @@ public extension AdyenScope where Base: UIViewController {
...
+                base.present(viewController, animated: animated, completion: completion)
```
Comment (mohammedDehairy) [inline Adyen/Helpers/UIViewControllerHelpers.swift:30]: this is wrong, its should be recursive

File: Demo/Common/IntegrationExample.swift [hunk 4]
```diff
@@ -77,8 +77,9 @@ internal final class IntegrationExample {
         let success = resultCode == .authorised || resultCode == .received || resultCode == .pending
 
         currentComponent?.stopLoading(withSuccess: success) { [weak self] in
-            self?.presenter?.dismiss(completion: nil)
-            self?.presentAlert(withTitle: resultCode.rawValue)
+            self?.presenter?.dismiss { [weak self] in
+                self?.presentAlert(withTitle: resultCode.rawValue)
```
Comment (descorp) [inline Demo/Common/IntegrationExample.swift:81]: This is what was causing alert presented on dismissed VC

**Review by mohammedDehairy — APPROVED** [review]

**Pull request merged**